main-app/MoveMaster-app/Assets/Scripts/Scoreboard/ScoreboardManager.cs

```diff
@@ -0,0 +1,43 @@
+using System.Collections;
+using System.Collections.Generic;
+using TMPro;
+using UnityEngine;
+
+public class ScoreboardManager : MonoBehaviour
+{
+    public static ScoreboardManager Instance;
+    [SerializeField]
+    private TMP_Text scorePoints;
+    [SerializeField]
+    private int score=0;
+
+    void Awake(){
+        if (Instance != null && Instance != this) 
+    { 
+        Destroy(this); 
+    } 
+    else 
+    { 
+        Instance = this; 
+    } 
+    }
+
+    void Start()
+    {
+        setScore(0);
+    }
+
+    public void setScore(int newScore){
```

Comment:
Funkcje z dużej litery (PascalCase) -> https://codenga.pl/strony/poradniki/konwencje_nazewnicze_w_jezyku_csharp?srsltid=AfmBOopii31SRMjnHHcPilzoQ0hr6TkAnyrZqN0qXTU8Cj0UdY4HKOJ2

Reply:
noo to już takie założenie, możemy się tego trzymać

Reply:
Do poprawy